action/lint.go

```diff
@@ -0,0 +1,160 @@
+package action
+
+import (
+	"github.com/google/go-github/github"
+	"github.com/helm/helm/chart"
+	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
+	"gopkg.in/yaml.v2"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
```

Comment:
Alpha order is also suggested. I usually just split the imports into two groups (core and other), and then let `go fmt` sort them for me.